### DISCUSSION TITLE
How would I add Google, Startpage and Brave search engines?

### DISCUSSION THREAD

**Discussion by Raj2032** [Eloston/ungoogled-chromium#2263] [Q&A]
![image](https://user-images.githubusercontent.com/29293830/222900099-efcf01cc-9807-4870-8e1c-2cbf25291466.png)

So I am trying to right now create startpage but not too sure what should be the 3rd and 4th box. I just want to know how it works?

**Comment by networkException** [comment]
See https://support.startpage.com/hc/en-us/articles/4520913488148-Search-strings-adding-Startpage-to-unsupported-web-browsers
